### ISSUE TITLE
Any way to change title bar size?

### ISSUE THREAD

**Issue by phimath** [tista500/Adapta#199]
**Reporter info**

 ```
 * Distribution - Ubuntu Gnome
 * Gtk+ 3.0 version - 3.18.9
 * Desktop environment -  Gnome Shell 3.18.5
 ```

**Adapta version (if you knew)**   3.21.4 (?)

**Related Application and/or Shell theming (or name of widget)**

**Actual issue**
I have a High DPI display and have Window Scaling set to 2.0 in Gnome Tweak Tool to compensate.  This makes everything look correct, but the Adapta Title Bars are massive!  Is there anyway to shrink the Title Bar icons during a build to fix this?  

**Steps to reproduce (if you knew)**

**Expected behaviour**

**Other Note (feature-request, question, etc...)**

Screen Size - 13.3"
Resolution - 3200 x 1800 (16:9)

Gnome Tweak Tool Settings:
Window Title Font - Cantarell Bold 11 (Although the window titles are actually using the interface font which is Cantarell Regular 11)
Scaling Factor - .80
Hinting - Slight

HiDPI Window Scaling - 2

**Comment by tista500** [comment]
Let me see the differences between Adapta and Adwaita.

**Comment by phimath** [comment]
Yeah, Adapta and Adwaita are very similar in size.  But themes like Arc, Paper and Vertex seem to handle High DPI displays really well, I was just hoping that you might consider adding similar support to Adapta.

**Comment by tista500** [comment]
> themes like Arc, Paper and Vertex seem to handle High DPI displays really well

I can't understand what was "well", So please explain more concrete things.
* LCD resolution and size
* Font scaling factor
* Font name
* what pixel did you see the differences compared with those themes

And you did NOT report your DEs and Gtk+ version, if not, how I track your issue down? tell me.

**Comment by phimath** [comment]
Sorry I have went back and filled in the missing information.

For reference, Adapta's Title bars for me are 79 pixels and Arc's are 57 (This is using an on screen ruler so I may be off by a pixel.)

Thanks for looking into this! I really love the theme and will likely keep using it even if the title bars are left as is. They just look massive on screen when next to things like Chrome tabs or Menu bars.

**Comment by tista500** [comment]
I've checked Arc as well,

![untitled](https://cloud.githubusercontent.com/assets/6073622/17790426/5b7d2786-65d1-11e6-98b9-f2266ce1db78.png)

Distribution: Ubuntu Gnome 16.04 LTS
Gtk+ version: 3.20.8
Desktop Environment: Gnome-Shell 3.20.3
Adapta version: Git latest https://github.com/tista500/Adapta/commit/a43ce0bf1d018fb0aa99d2a5a57763128d139f3e

LCD: 11.6" QHD (2560x1440)
Machine: Dell XPS 11 (9P33)

Window scaling factor: 2.0
Font scaling factor: 0.8
Font for interface: Noto Sans regular
Font size: 10pt

Arc: 58px height
Adapta: 68px height
Exactly 10px difference existed. But I will not reduce such 10px anymore. On 13.3" and/or 14" with native QHD, QHD+, UHD resolution, I think 34px is the smallest value for touchscreen LCDs without stylus.

Regards.

**Comment by kieranlea** [comment]
Assuming this is what you're talking about, I think it depends on the program. I have noticed massive title bars on my hidpi display too but only when the icons are square. Apps with circle material icons like nautilus look great. 

Example is firefox:
![screenshot from 2016-08-18 22-23-10](https://cloud.githubusercontent.com/assets/9081619/17796951/9aaf99dc-6592-11e6-9c7d-42ed2a431683.png)

**Comment by tista500** [comment]
@phimath,

I've also checked Gtk+ 3.18.9 & Gnome-Shell 3.18.5 on my Thinkpad X1 Carbon 2015 (3rd-Gen). It seems to keep 66 or 67px in various combinations of font-scalings/font-sizes...

> Adapta's Title bars for me are 79 pixels and Arc's are 57

This commit https://github.com/tista500/Adapta/commit/3ee5dd120979b568f3a5b4fe9d542bd970cc90d9 reduced the height about 5 or 6px at native resolution, and 10 or 11px at 'x2' scaled resolution, So I think this issue was solved...

Regards.

@kieranlea,

Hmm, I don't think your shot was related to this issue. And your shot was not related to theme-specific problem as well. Maybe your system has "mixed versioned" GNOME environment, like the combo 3.20.`libgtk-3.0` and 3.18.x `libmutter`.  See this: https://github.com/tista500/Adapta/issues/188. That issue was happened on Ubuntu Yakkety unfortunately, but it could happen in any distributions those have such mixed (ugly) versioning within GNOME core libraries...

Regards.
